test/Writer/FeedTest.php

```diff
@@ -180,6 +181,14 @@ public function testSetDateCreatedUsesDateTimeObject()
         $this->assertEquals($myDate, $writer->getDateCreated());
     }
 
+    public function testSetDateCreatedUsesDateTimeImmutableObject()
+    {
+        $myDate = new DateTimeImmutable('@' . 1234567890);
```

Comment:
We should definitely use data-providers for the entire test class – _later_!

Reply:
Hell yeah!